pkg/tbtc/chain_test.go

```diff
@@ -129,74 +143,291 @@ func (lc *localChain) OnDKGResultSubmitted(
 	})
 }
 
-// SubmitDKGResult submits the DKG result to the chain, along with signatures
-// over result hash from group participants supporting the result.
+func (lc *localChain) OnDKGResultChallenged(
+	handler func(event *DKGResultChallengedEvent),
+) subscription.EventSubscription {
+	lc.dkgResultChallengeHandlersMutex.Lock()
+	defer lc.dkgResultChallengeHandlersMutex.Unlock()
+
+	handlerID := local_v1.GenerateHandlerID()
```

Comment:
What do you think about copy-pasting `GenerateHandlerID` to `pkg/tbtc/chain_test.go` so that we are revering `local_v1` only when absolutely necessary? This should make future refactoring easier.